containers/webserver/config/shopware/default.conf

```
@@ -5,7 +5,7 @@
 
 	DocumentRoot /var/www
 	<Directory /var/www/>
-		Options -Indexes +SymLinksIfOwnerMatch -MultiViews
+		Options +ExecCGI +FollowSymlinks -SymLinksIfOwnerMatch
```

I thinks we have the same folder permissions problem here like we have in neos bevor.
Could you try out this settings:
```suggestion
		Options -Indexes +FollowSymLinks -SymLinksIfOwnerMatch -MultiViews
```